frontend/packages/ux-editor/src/containers/DesignView/AddItem/AddItemContent.module.css

```diff
@@ -6,6 +6,13 @@
   flex: 3;
   padding: 20px;
   overflow-y: scroll;
+  gap: var(--fds-spacing-4);
```

Comment:
As we are transitioning to the new version of `Designsystemet` we should use the new CSS variables to avoid technical debt 😄